packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -1,9 +1,10 @@
 import type { AuthObject } from '@clerk/backend/internal';
+import { notFound, redirect } from 'next/navigation';
```

Comment:
❓ Are those exports supported in the minimum NextJS version defined in our package.json?

Reply:
Yes :)